ipaserver/dcerpc.py

```diff
@@ -58,7 +58,7 @@
 import pysss_nss_idmap
 import pysss
 import six
-from ipaplatform.paths import paths
+from ipaplatform.paths import paths  # pylint: disable=E0401
```

Comment:
Please use message ids (`import-error`) rather than message codes (`E0401`) in pylint comments.